types/scheduler.go

```diff
@@ -44,7 +48,7 @@ func (s *Scheduler) AddCronFunction(c CronFunction, invoker *types.Invoker) (Sch
 // NewScheduler returns a scheduler
 func NewScheduler() *Scheduler {
 	return &Scheduler{
-		cron.New(),
+		cron.New(cron.WithParser((standardParser))),
```

Comment:
Should `cron.WithParser((standardParser))` be `cron.WithParser(standardParser)` ?